src/NzbDrone.Core/Datastore/Migration/213_add_api_key_to_users.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(213)]
+    public class add_api_key_to_users : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Users").AddColumn("ApiKey").AsString().WithDefaultValue("Test");
```

Comment:
Since the only user would be the admin user when this runs it should set the API key that already exists.